tools/mo/openvino/tools/mo/moc_frontend/extractor.py

```diff
@@ -227,6 +227,21 @@ def fe_input_user_data_repack(
     else:
         assert input_user_shapes is None
     # TODO: implement freeze_placeholder (issue 58560)
+    # At the moment set_tensor_value is implemented only for onnx
+    if framework == 'onnx' and freeze_placeholder is not None:
+        for name, value in freeze_placeholder.items():
+            node = decode_name_with_port(input_model, name, framework, IOType.Input)
+            if not node.is_input():
+                raise Error(f"{name} is not an input node. Only input nodes can be freezed!")
+            if input_user_data_types.get(name) is not None:
+                value = np.array(value, dtype=input_user_data_types[name])
+            elif value in ['True', 'False']:
+                value = np.array(value, dtype=np.bool)
+            else:
+                value = np.array(value, dtype=np.float32)
+            # Only input layer
+            input_model.set_tensor_value(node, value)
```

Comment:
Actually, user can specify freezing value of any supported type and shape. Please read MO help about it.
For example, user can freeze `node1:0` to `i32` tensor of shape `[1 2 3]` with values `[1 2 3 4 5 6]`:
```
--input "node1:0[1 2 3]{i32}->[1 2 3 4 5 6]"
```
Also, MO supports other types. Please handle it

Reply:
please add more test cases for this